### DISCUSSION TITLE
Backup einspielen

### DISCUSSION THREAD

**Discussion by Pflegecraft** [ophian/styx#29] [Q&A] [answered]
Hallo wie kann ich von dem Backup Plugin die Datenbank wieder einspielen?
Meine Datei heißt 2022-01-08-11-01_sqlbackup.tar.gz aber wie spiele ich diese in Myphpadmin wieder ein?

Bekomme jedesmal eine Error Meldung:
Fehler
Statische Analyse:

1713 Fehler wurden während der Analyse gefunden.

**Comment by ophian** [comment]
*.tar.gz ist ein "zip" artiges komprimierungs Format. Das muss man erst auspacken damit man zur eigentlichen *.sql Datei kommt, die man dann zum wiedereinspielen nutzen kann. https://mizine.de/linux/targz-entpacken/

**Reply by ophian** [reply]
das Backup Plugin ist ein sehr altes Plugin und nur notdürftig an heutige Bedürfnisse angepasst, aber (wenigstens von mir) noch nie vollständig durchgetestet, so dass ich die Benutzung empfehlen könnte.  Man sollte immer ein vollständiges Backup mit unabhängigen Prozeduren (phpMyAdmin und eigen erstelltes zip) haben bevor man sich darauf wirklich verlässt.

**Reply by ophian** [reply]
Plugin Backup 1.2.0 (gerade released) sollte nun wieder durchlaufen. (jedenfalls "annährend" getestet)
Ohne eigene und unabhängige Zips aber würde ich mich nicht daran wagen... jedenfalls nicht auf einem LIVE system!

**Reply by Pflegecraft** [reply]
Jo es ging trotzdem nicht.. ist egal.. ich mache meinen Blog manuell wieder startklar :D waren ja nur 23 Beiträge....

**Reply by ophian** [reply]
Was heißt das? Was ging wie genau nicht? Unterstützt dein Server etwas nicht oder gab es konkrete Fehler?
Ansonsten sind in der der `abc.tar.gz` Datei alle SQL files per Tabelle einzeln abglegt, also zb für entries als `2022-01-27-12-25_INSERT_s9y_entries.sql`. Ein Program wie [7z](https://7-zip.org/) unter Windows kann das öffnen, damit man sie gegebenenfalls mit PhpMyAdmin auch selbst manuell importieren kann.

**Reply by Pflegecraft** [reply]
Ich danke dir ich habe es hinbekommen der Blog läuft wieder und das sogar nun auch unter php 8.1 und das sogar super :)
Vielen dank für deine mühe in dieses CMS und deinen klasse Support.

**Comment by Pflegecraft** [comment]
Nach dem einspielen des Backups. bekomme ich jetzt eine 500er Meldung
Error.los sagt dies.
`[Sun May 01 11:00:36.855791 2022] [proxy_fcgi:error] [pid 922:tid 139978381887232] [client 162.158.203.36:42028] AH01071: Got error 'PHP message: PHP Fatal error:  Uncaught RuntimeException: The file could not be opened. Check permissions. in /home/users/pflegecraft/www/blog/bundled-libs/katzgrau/klogger/src/Logger.php:134\nStack trace:\n#0 /home/users/pflegecraft/www/blog/include/functions.inc.php(1547): Katzgrau\\KLogger\\Logger->__construct()\n#1 /home/users/pflegecraft/www/blog/serendipity_config.inc.php(364): serendipity_initLog()\n#2 /home/users/pflegecraft/www/blog/index.php(11): include('...')\n#3 {main}\n  thrown in /home/users/pflegecraft/www/blog/bundled-libs/katzgrau/klogger/src/Logger.php on line 134', referer: https://pflegecraft.de/
[Sun May 01 11:00:43.143289 2022] [proxy_fcgi:error] [pid 922:tid 139978348316416] [client 162.158.203.36:42038] AH01071: Got error 'PHP message: PHP Fatal error:  Uncaught RuntimeException: The file could not be opened. Check permissions. in /home/users/pflegecraft/www/blog/bundled-libs/katzgrau/klogger/src/Logger.php:134\nStack trace:\n#0 /home/users/pflegecraft/www/blog/include/functions.inc.php(1547): Katzgrau\\KLogger\\Logger->__construct()\n#1 /home/users/pflegecraft/www/blog/serendipity_config.inc.php(364): serendipity_initLog()\n#2 /home/users/pflegecraft/www/blog/index.php(11): include('...')\n#3 {main}\n  thrown in /home/users/pflegecraft/www/blog/bundled-libs/katzgrau/klogger/src/Logger.php on line 134', referer: https://pflegecraft.de/
[Sun May 01 11:01:34.605292 2022] [proxy_fcgi:error] [pid 922:tid 139978381887232] [client 172.70.251.162:50340] AH01071: Got error 'PHP message: PHP Fatal error:  Uncaught RuntimeException: The file could not be opened. Check permissions. in /home/users/pflegecraft/www/blog/bundled-libs/katzgrau/klogger/src/Logger.php:134\nStack trace:\n#0 /home/users/pflegecraft/www/blog/include/functions.inc.php(1547): Katzgrau\\KLogger\\Logger->__construct()\n#1 /home/users/pflegecraft/www/blog/serendipity_config.inc.php(364): serendipity_initLog()\n#2 /home/users/pflegecraft/www/blog/index.php(11): include('...')\n#3 {main}\n  thrown in /home/users/pflegecraft/www/blog/bundled-libs/katzgrau/klogger/src/Logger.php on line 134'
[Sun May 01 11:01:44.931931 2022] [proxy_fcgi:error] [pid 922:tid 139978348316416] [client 162.158.203.36:42104] AH01071: Got error 'PHP message: PHP Fatal error:  Uncaught RuntimeException: The file could not be opened. Check permissions. in /home/users/pflegecraft/www/blog/bundled-libs/katzgrau/klogger/src/Logger.php:134\nStack trace:\n#0 /home/users/pflegecraft/www/blog/include/functions.inc.php(1547): Katzgrau\\KLogger\\Logger->__construct()\n#1 /home/users/pflegecraft/www/blog/serendipity_config.inc.php(364): serendipity_initLog()\n#2 /home/users/pflegecraft/www/blog/index.php(11): include('...')\n#3 {main}\n  thrown in /home/users/pflegecraft/www/blog/bundled-libs/katzgrau/klogger/src/Logger.php on line 134', referer: https://pflegecraft.de/
[Sun May 01 11:02:11.184786 2022] [proxy_fcgi:error] [pid 922:tid 139978348316416] [client 172.70.34.181:60490] AH01071: Got error 'PHP message: PHP Fatal error:  Uncaught RuntimeException: The file could not be opened. Check permissions. in /home/users/pflegecraft/www/blog/bundled-libs/katzgrau/klogger/src/Logger.php:134\nStack trace:\n#0 /home/users/pflegecraft/www/blog/include/functions.inc.php(1547): Katzgrau\\KLogger\\Logger->__construct()\n#1 /home/users/pflegecraft/www/blog/serendipity_config.inc.php(364): serendipity_initLog()\n#2 /home/users/pflegecraft/www/blog/index.php(11): include('...')\n#3 {main}\n  thrown in /home/users/pflegecraft/www/blog/bundled-libs/katzgrau/klogger/src/Logger.php on line 134'
[Sun May 01 11:02:26.714221 2022] [proxy_fcgi:error] [pid 922:tid 139978348316416] [client 172.70.175.194:21992] AH01071: Got error 'PHP message: PHP Fatal error:  Uncaught RuntimeException: The file could not be opened. Check permissions. in /home/users/pflegecraft/www/blog/bundled-libs/katzgrau/klogger/src/Logger.php:134\nStack trace:\n#0 /home/users/pflegecraft/www/blog/include/functions.inc.php(1547): Katzgrau\\KLogger\\Logger->__construct()\n#1 /home/users/pflegecraft/www/blog/serendipity_config.inc.php(364): serendipity_initLog()\n#2 /home/users/pflegecraft/www/blog/index.php(11): include('...')\n#3 {main}\n  thrown in /home/users/pflegecraft/www/blog/bundled-libs/katzgrau/klogger/src/Logger.php on line 134'
[Sun May 01 11:02:41.199118 2022] [proxy_fcgi:error] [pid 922:tid 139978348316416] [client 162.158.203.36:42176] AH01071: Got error 'PHP message: PHP Fatal error:  Uncaught RuntimeException: The file could not be opened. Check permissions. in /home/users/pflegecraft/www/blog/bundled-libs/katzgrau/klogger/src/Logger.php:134\nStack trace:\n#0 /home/users/pflegecraft/www/blog/include/functions.inc.php(1547): Katzgrau\\KLogger\\Logger->__construct()\n#1 /home/users/pflegecraft/www/blog/serendipity_config.inc.php(364): serendipity_initLog()\n#2 /home/users/pflegecraft/www/blog/index.php(11): include('...')\n#3 {main}\n  thrown in /home/users/pflegecraft/www/blog/bundled-libs/katzgrau/klogger/src/Logger.php on line 134', referer: https://pflegecraft.de/
`

**Comment by ophian** [comment, marked as answer]
Immer dasselbe für jeden Schreibvorgang: "_The file could not be opened. Check permissions._"
Ich nehme mal an du hast Log Level angeschaltet.
Check mal deine permissions für `templates_c` und `templates_c/logs` Das Verzeichnis `logs` muss ebenfalls (be)schreibbar sein.
